skyportal/handlers/api/sources.py

```diff
@@ -789,136 +790,141 @@ async def get_sources(
                     f'Invalid created_or_modified_after: {created_or_modified_after} ({e})'
                 )
 
-        # PHOTSTATS
-        photstat_query = []
-        if first_detected_date is not None:
-            try:
-                col = (
-                    'first_detected_mjd'
-                    if not exclude_forced_photometry
-                    else 'first_detected_no_forced_phot_mjd'
-                )
-                query_params.append(
-                    bindparam(
-                        'first_detected_date',
-                        value=Time(arrow.get(first_detected_date).datetime).mjd,
-                        type_=sa.Float,
+        if require_detections:
```

Comment:
do we really need an extra boolean? Can't we just not make it mandatory to specify first and last detection dates and pop it out of the form in the frontend? Seems like a very confusing way to handler parameters when we have one parameter that prevents other parameters from being used. Especially confusing for someone using the API. I would just not make the arguments mandatory when querying a localization or smth.

Reply:
I am ok with that as well. I do think the nominal behavior should be to encourage checking photometry and this does do that. But maybe a better frontend interface might suffice

Reply:
Let me make some changes to that PR through this week, and I'll ask for a review!

But thanks a lot for all your help with the different requests from Mansi's group

Reply:
Ah and also the changes need to be made to the `get_sources` method from the `sources.py` file, not this one which is being deprecated (and not the one used by the frontend anymore). I'm planning on deleting the old one soon.

Reply:
You do make a point about encouraging folks to include first and last detection, unless they pass an argument to explicitly not constrain on photometry. I'll keep that behavior and just mirror it in the new `get_sources` method too.

Reply:
Ah but you did that too!!! One week off and my brain cells are dead. Then I'm happy with this, just pushing a minor frontend change and we can merge